public/app/features/admin/ldap/LdapDrawer.tsx

```diff
@@ -0,0 +1,251 @@
+import { css } from '@emotion/css';
+import { useFormContext } from 'react-hook-form';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import {
+  useStyles2,
+  CollapsableSection,
+  Divider,
+  Drawer,
+  Field,
+  Icon,
+  Input,
+  Label,
+  Select,
+  Stack,
+  Switch,
+  Text,
+  TextLink,
+  Tooltip,
+} from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+import { LdapPayload } from 'app/types';
+
+interface Props {
+  onClose: () => void;
+}
+
+const tlsOptions: Array<SelectableValue<string>> = ['TLS1.2', 'TLS1.3'].map((v) => ({ label: v, value: v }));
+
+export const LdapDrawerComponent = ({ onClose }: Props) => {
+  const styles = useStyles2(getStyles);
+  const { register, setValue, watch } = useFormContext<LdapPayload>();
+
+  const groupMappingsLabel = (
+    <Label
+      className={styles.sectionLabel}
+      description={t('ldap-drawer.group-mapping-section.description', 'Map LDAP groups to Grafana org roles')}
+    >
+      <Trans i18nKey="ldap-drawer.group-mapping-section.label">Group mapping</Trans>
+    </Label>
+  );
+
+  const useTlsDescription = (
+    <Trans i18nKey="ldap-drawer.extra-security-section.use-ssl.tooltip">
+      For a complete list of supported ciphers and TLS versions, refer to:{' '}
+      {
+        <TextLink style={{ fontSize: 'inherit' }} href="https://go.dev/src/crypto/tls/cipher_suites.go" external>
+          https://go.dev/src/crypto/tls/cipher_suites.go
+        </TextLink>
+      }
+    </Trans>
+  );
+
+  return (
+    <Drawer title={t('ldap-drawer.title', 'Advanced Settings')} onClose={onClose}>
```

Comment:
think we use sentence case everywhere, so this should be `Advanced settings`. haven't checked all the other strings, there might be others